Resources/Locale/en-US/accent/french.ftl

```diff
@@ -0,0 +1,131 @@
+accent-french-replaced-1 = yes
+accent-french-replacement-1 = oui
+
+accent-french-replaced-2 = hi
+accent-french-replacement-2 = salut
+
+accent-french-replaced-3 = hello
+accent-french-replacement-3 = allo
+
+accent-french-replaced-4 = bye
+accent-french-replacement-4 = au revoir
+
+accent-french-replaced-5 = goodbye
+accent-french-replacement-5 = au revoir
+
+accent-french-replaced-6 = the
+accent-french-replacement-6 = la
+
+accent-french-replaced-7 = is
+accent-french-replacement-7 = est
+
+accent-french-replaced-8 = bread
+accent-french-replacement-8 = baguette
+
+accent-french-replaced-9 = very
+accent-french-replacement-9 = très
+
+accent-french-replaced-10 = good
+accent-french-replacement-10 = bien
+
+accent-french-replaced-11 = great
+accent-french-replacement-11 = super
+
+accent-french-replaced-12 = beautiful
+accent-french-replacement-12 = beau
+
+accent-french-replaced-13 = music
+accent-french-replacement-13 = musique
+
+accent-french-replaced-14 = greetings
+accent-french-replacement-14 = bonjour
+
+accent-french-replaced-15 = france
+accent-french-replacement-15 = la france
+
+accent-french-replaced-16 = captain
+accent-french-replacement-16 = capitaine
+
+accent-french-replaced-17 = fries
+accent-french-replacement-17 = frites
+
+accent-french-replaced-18 = wand
+accent-french-replacement-18 = baguette
+
+accent-french-replaced-19 = magic
+accent-french-replacement-19 = magie
+
+accent-french-replaced-20 = magic wand
+accent-french-replacement-20 = baguette magique
+
+accent-french-replaced-21 = we
+accent-french-replacement-21 = nous
+
+accent-french-replaced-22 = small
+accent-french-replacement-22 = petite
+
+accent-french-replaced-23 = thanks
+accent-french-replacement-23 = merci
+
+accent-french-replaced-24 = thankyou
+accent-french-replacement-24 = merci
```

Comment:
Why is this missing a space? Are you trying to catch typos too? This is overdoing it.